wildlifelicensing/apps/main/forms.py

```diff
@@ -105,34 +106,41 @@ def __init__(self, *args, **kwargs):
                 field.required = False
         else:
             # enforce required for some fields not required at the ledger (Licence) model level
-            required = ['issue_date', 'start_date', 'end_date']
+            required = ['start_date', 'end_date']
             for field_name in required:
                 field = self.fields.get(field_name)
                 if field is not None:
                     field.required = True
 
-        if purpose is not None:
-            self.fields['purpose'].initial = purpose
-
         self.fields['is_renewable'].widget = forms.CheckboxInput()
 
-        # if a licence instance has not been passed in nor any POST data
-        # (i.e. this is creating the 'get' version of the form)
+        # if a licence instance has not been passed in nor any POST data (i.e. get version of form)
         if 'instance' not in kwargs and len(args) == 0:
-            today_date = datetime.now()
-            self.fields['issue_date'].initial = today_date.strftime(DATE_FORMAT)
-            self.fields['start_date'].initial = today_date.strftime(DATE_FORMAT)
+            today_date = date.today()
 
-            self.fields['issue_date'].localize = False
+            self.fields['start_date'].initial = today_date.strftime(DATE_FORMAT)
 
-            if default_period is not None:
+            if end_date is not None:
+                self.fields['end_date'].initial = end_date
+            elif default_period is not None:
                 end_date = today_date + relativedelta(days=default_period)
 
                 self.fields['end_date'].initial = end_date.strftime(DATE_FORMAT)
 
+            self.fields['regions'].initial = regions
             self.fields['is_renewable'].initial = is_renewable
-
             self.fields['return_frequency'].initial = return_frequency
+            self.fields['locations'].initial = locations
+            self.fields['purpose'].initial = purpose
+            self.fields['additional_information'].initial = additional_information
+            self.fields['cover_letter_message'].initial = cover_letter_message
+
+    def clean_start_date(self):
+        start_date = self.cleaned_data['start_date']
+        if start_date < date.today():
+            raise forms.ValidationError('Start date cannot before current date')
+

```

Comment:
This rule is only for new licences, not amended or reissued (see Walter [comment](https://kanboard.dpaw.wa.gov.au/?controller=TaskViewController&action=show&task_id=3411&project_id=24))